package/MDAnalysis/lib/formats/libdcd.pyx

```diff
@@ -0,0 +1,706 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
```

Comment:
This is the old header!

Please update to the one in eg `MDAnalysis/__init__.py`.